models/webhooks_events.php

```diff
@@ -57,6 +59,11 @@ public function getObservers()
     {
         // Check if the current instance of the class, has the observers stored in memory
         if (!empty($this->observers) && is_array($this->observers)) {
+            Cache::clearCache(
```

Comment:
This makes the cache essentially useless.  It'll clear it every time this function is called.

One compromise is to only cache system events and fetch the plugin ones each time. Also clear the system cache every once in a while.  Maybe a daily cron.

Reply:
Perhaps even better, we could store/cache a list of installed plugins, then only clear the observer cache if the list of installed plugins has changed.